utils/storage_utils.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {GlobalState} from 'types/store';
+import {GlobalState} from '@mattermost/types/store';
```

Comment:
Narrowed the type because the wider global state type wasn't needed.

Reply:
I'm surprised that's possible, but I guess this doesn't use `state.storage` at all